Sources/ComposableArchitecture/Documentation.docc/Articles/MigrationGuides/MigratingTo1.11.md

```diff
@@ -62,7 +62,7 @@ To fix this deprecation you can use the new ``Shared/withLock(_:)`` method on th
 case .delayedIncrementButtonTapped:
   return .run { _ in
     @Shared(.count) var count
-    $count.withLock { $0 += 1 }
+    await $count.withLock { $0 += 1 }
```

Comment:
@mbrandonw I can, but looks too small change for PR.

Reply:
This PR was already merged, that is why you can't suggest a change.